app/assets/javascripts/views/data_portal/ChartWidgetView.js

```diff
@@ -12,13 +12,13 @@
     url: function() {
       var url =  API_URL + '/indicator/' + this.indicatorId + '?' + this.iso + '=' + this.year;
 
-      if (this.filters.length) {
+      if (this.filters.length >= 1 && !_.isEmpty(this.filters[0])) {
```

Comment:
This bit, I think, is caused by the empty object in `this.options._filters` (see one of the previous comments). I guess we could remove this.